Core/src/main/java/com/paypal/android/core/API.kt

```diff
@@ -14,4 +14,17 @@ class API internal constructor(
             httpRequestFactory.createHttpRequestFromAPIRequest(apiRequest, configuration)
         return http.send(httpRequest)
     }
+
+    suspend fun getClientId(): String {
+        val apiRequest = APIRequest(
+            path = "v1/oauth2/token",
+            method = HttpMethod.GET,
+            body = null
+        )
+        val httpRequest =
+            httpRequestFactory.createHttpRequestFromAPIRequest(apiRequest, configuration)
+        val response = http.send(httpRequest)
+        val json = PaymentsJSON(response.body!!)
```

Comment:
throw error when body is null?
` if (bodyResponse.isNullOrBlank()) {
                throw APIClientError.noResponseData(correlationID)
            }`

Reply:
Also a good call! I added in some error handling logic with tests.